src/hooks/useUpdateLayoutEffect.ts

```diff
@@ -0,0 +1,22 @@
+import { useLayoutEffect, useRef } from 'react'
+
+/**
+ * A custom useEffect hook that only triggers on updates, not on initial mount
+ * Idea stolen from: https://stackoverflow.com/a/55075818/1526448
+ * @param {()=>void} effect the function to call
+ * @param {Array<any>} dependencies the state(s) that fires the update
+ */
+export function useUpdateLayoutEffect(
+  effect: () => void,
+  dependencies: any[] = [],
+) {
+  const isInitialMount = useRef(true)
+
+  useLayoutEffect(() => {
+    if (isInitialMount.current) {
+      isInitialMount.current = false
+    } else {
+      effect()
+    }
+  }, dependencies) // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
we use tslint, so eslint disable comment is not relevant

Reply:
sorry, this is also a relic from my project, i'll take it out.